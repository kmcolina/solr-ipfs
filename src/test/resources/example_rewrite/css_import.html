<html>
<head>
    <title>CSS includes</title>
    <style>
        @import url('someinclude-1b5761d5e5bc48746b24cb1d708223cd-blessed1.css?z=1384381230968_o1');
        @import url(slidearrows_o2.css);
        @import url(shadow_frames_o3.css);
    </style>
</head>
<h1>Multiple CSS includes</h1>

</html>
