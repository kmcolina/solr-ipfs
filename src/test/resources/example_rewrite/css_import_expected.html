<html>
 <head>
  <title>CSS includes</title>
  <style>
 @import url('http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&offset=1');
 @import url(http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&offset=2);
 @import url(http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&offset=3);
 </style>
 </head>
 <body>
  <h1>Multiple CSS includes</h1>
 </body>
</html>